### PR TITLE
feat: contact layout update

### PR DESCRIPTION
**Pull request by imklau**
## Description 📝

New Contact page layout, and new founder asset

## Related Issues 🔗

JIRA issue: [FSU-435](https://klau.atlassian.net/browse/FSU-435)

## Preview Links 🔍

- [/kontakt](https://sterczaceuszy-git-fsu-435-non-profit-dev.vercel.app/kontakt)

### CODE REVIEW

**Review by agnieszkastanczuk — APPROVED** [review]

File: packages/frontend/app/templates/Contact/Contact.jsx [hunk 1]
```diff
@@ -30,16 +34,26 @@ const Contact = () => {
...
-          )}
-        </Styled.ContactContainer>
-      </Container>
+      <main>
+        <Hero
+          heading="Bądźmy w kontakcie!"
+          subtitle="Masz pytania? Pisz śmiało!"
+          description="Jeśli masz jakiekolwiek pytania nie wahaj się pisać lub wysłać smsa. Odpowiadamy tak szybko jak to możliwe."
+          backgroundColor={theme.colors.complementary[100]}
+          asset="/contact/Hero.png"
```
Comment (agnieszkastanczuk) [inline packages/frontend/app/templates/Contact/Contact.jsx:43]: asset="/contact/hero.png"

File: packages/frontend/app/templates/Contact/components/ContactInfo/ContactInfo.jsx [hunk 2]
```diff
@@ -25,10 +24,17 @@ const ContactInfo = () => {
...
+
+      <Styled.FounderInfo>
+        <Styled.FounderImage src="/contact/VanessaSzmatuła.png" alt="" />
+        <Typography variant="h5" as="p">
+          Vanessa Szmatuła
+        </Typography>
+      </Styled.FounderInfo>
+
+      <List gap={14}>
         {listData.map(({ text, iconName, link }) => (
```
Comment (paulgrym) [inline packages/frontend/app/templates/Contact/components/ContactInfo/ContactInfo.jsx:38]: there is also a second phone number in the design

**Review by paulgrym — APPROVED** [review]

File: packages/frontend/app/templates/Contact/components/ContactInfo/ContactInfo.jsx (hunk 2 above)
Comment (imklau) [reply packages/frontend/app/templates/Contact/components/ContactInfo/ContactInfo.jsx:38]: the second one was removed recently - https://github.com/non-profit-dev/sterczace-uszy/pull/220 (design isn't up to date)

File: packages/frontend/app/templates/Contact/Contact.jsx (hunk 1 above)
Comment (imklau) [inline packages/frontend/app/templates/Contact/Contact.jsx:43]:
```suggestion
          asset="/contact/hero.png"
```

**Pull request merged**